sweagent/agent/models.py

```diff
@@ -326,7 +327,7 @@ def history_to_messages(
         Reference: https://docs.anthropic.com/claude/reference/complete_post
         """
         # Preserve behavior for older models
-        if self.api_model in ["claude-instant", "claude-2"]:
+        if self.api_model in ["claude-instant", "claude-2", "claude-2.1"]:
```

Comment:
This may be Bug. If not specified here, it will call the API of the Claude3 family.

Reply:
Agreed! Let me push a fix to main already, just so this doesn't cause any issue! Thank you!

Reply:
Oh, let me actually push a fix to this right now then!

Reply:
Thank you for fixing it right away!